README.md

```diff
@@ -127,3 +128,22 @@ The code includes the following components:
 
 <p><img src="./docs/converting-to-mlrun-pipeline.png" alt="converting-to-mlrun pipeline output"/></p>
 
+<a id="stock-prediction"></a>
+## Stocks prices prediction demo
+
+The [**stock-prediction**](stock-prediction/README.md) demo demonstrates how to use feature store to create two ingestion services - news and stocks, news pipeline enriches the stocks news scraped from [yahoo-api](https://finance.yahoo.com/) with its sentiments using a pretrained sentiment analysis model.<br>
```

Comment:
The [**stock-prediction**](stock-prediction/README.md) demo demonstrates how to use feature store to create two ingestion services - news and stocks. News....